public/timeline-report.js

```diff
@@ -14,8 +14,8 @@ import "./select-report-type/select-report-type.js";
 import "./select-view-settings/select-view-settings.js";
 
 import { rollupAndRollback } from "./jira/rolledup-and-rolledback/rollup-and-rollback";
-import { calculateReportStatuses } from "./jira/rolledup/work-status.js/work-status.js";
-import { groupIssuesByHierarchyLevelOrType } from "./jira/rollup/rollup.js";
+import { calculateReportStatuses } from "./jira/rolledup/work-status.js/work-status";
+import { groupIssuesByHierarchyLevelOrType } from "./jira/rollup/rollup";
 import { pushStateObservable } from "./shared/state-storage.js";
```

Comment:
had a merge conflict here, i removed the .js extensions and main added pushStateObservable, used main and re-removed the extensions